analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -82,7 +82,7 @@ object VirtualEnv : CommandLineTool {
     override fun command(workingDir: File?) = "virtualenv"
 
     // Allow to use versions that are known to work. Note that virtualenv bundles a version of pip.
-    override fun getVersionRequirement(): Requirement = Requirement.buildIvy("[15.1,16.7[")
+    override fun getVersionRequirement(): Requirement = Requirement.buildIvy("[15.1,)")
```

Comment:
I actually liked having a place where I can look-up the most recent version known to be working with `ORT` which this place tries to eliminate. That is maybe a use case when some user wants to try ORT outside docker and go installing its deps.

I'm ok with adjusting the check like this, even if I had a slight preference for [15.1, 17.0[
If it remains fully unbounded would you be fine adding a comment saying that it works with 16.x or 16.7?

Reply:
> I'm ok with adjusting the check like this, even if I had a slight preference for [15.1, 17.0[

Alright, I went for the latter.

Reply:
Thanks!